src/main/java/com/ibanity/apis/client/products/ponto_connect/services/impl/TransactionServiceImpl.java

```diff
@@ -46,17 +47,25 @@ public IbanityCollection<Transaction> list(TransactionsReadQuery transactionsRea
             pagingSpec = IbanityPagingSpec.DEFAULT_PAGING_SPEC;
         }
 
-        URI uri = buildUri(getUrl(transactionsReadQuery.getAccountId()), pagingSpec);
+        URI uri = buildUri(getUrl(transactionsReadQuery.getAccountId(), transactionsReadQuery.getSynchronizationId()), pagingSpec);
```

Comment:
Maybe worth creating a new method called listUpdatedTransactions or listUpdatedTransactionsForSynchronization like you named it in the unit test + specific query parameter,  so it's clear for the developer that it's a subset of transactions.

Reply:
Change made. Is this what you had in mind? If it's good like this I'll make the same change on XS2A for consistency.

Reply:
Yes cool :+1: